README.md

```diff
@@ -1,16 +1,15 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Portfolio - Jorum Odiemo
 
-> Description the project.
+> Creating a skeleton mobile website of my portfolio
+> 
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
+- HTML & CSS3
+- Flexbox and Grid
 ## Live Demo (if available)
```

Comment:
- [ ] As the previous reviewer mentioned, kindly customize the README. you can remove the unused headers/sections, from the line `18` to line `36`
```
## Getting Started
**This is an example of how you may give instructions on setting up your project locally.**
**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
To get a local copy up and running follow these simple example steps.
### Prerequisites
### Setup
### Install
### Usage
### Run tests
### Deployment
```